pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/VirtualEnvironment.java

```diff
@@ -51,16 +51,20 @@ public static String getPythonApplicationDirectory() {
         return OperatingSystem.current().isWindows() ? "Scripts" : "bin";
     }
 
-    public File getExecutable(String path) {
-        return validateFileExists(prefixBuilder().resolve(OperatingSystem.current().getExecutableName(path)));
+    public File getExecutable(String execName) {
```

Comment:
The type is the same here (String), but the semantics is not (name vs. path).
Are we sure that existing usage of this method and the `findExecutable` below will be OK with this change in semantics?

Reply:
It was never really a path, it was always something like "pip" or "pex". So I think the name change is better. Pip and Pex were found during tests/integTests so I think it will be OK.